src/specific/memory.c

```diff
@@ -9,7 +9,6 @@
 
 static char *m_GameMemoryPointer = NULL;
 static char *m_GameAllocMemPointer = NULL;
-static uint32_t m_GameAllocMemUsed = 0;
```

Comment:
I feel a better use of this is to keep it and throw asserts or a S_ExitSystemFmt on it, so we actually check we don't run out of RAM. While at 16MB no original level is going to, but TRLE mappers be crazy.

Reply:
We can infer this value directly from m_GameAllocMemFree + we already check for memory exhaustion.

Reply:
yeah, missed that. I guess it was originally for a screen print.